style.css

```diff
@@ -0,0 +1,990 @@
+
+  .section-desc__wrap-down { background-position: 99% 23%; }
+
+  .safe-body__item_corps {
+    top: 200px;
+    right: 40px;
+  }
+}
+
+@media (max-width: 1032px) {
+  .text__content { width: 97%; }
+
+  .text__wrap_character-down,
+  .text__wrap_down { padding-left: 20px; }
+  .text__wrap_character-down { height: 205px; }
+
+  .score {
+    padding-left: 21px;
+  }
+}
+
+@media (max-width: 900px) {
+  .text__wrap_character-down,
+  .text__wrap_down { padding-left: 8px; }
+
+  .text__title_down { padding-left: 0; }
+
+  .score { padding-left: 15px; }
+
+  .button__item_big-down { padding-left: 11px; }
+
+  .safe-body__item_corps {
+    top: 260px;
+    right: 32px;
+  }
+
+  .section-desc__wrap-down { background-position: 99% 31%; }
+
+  .item__icon_corps { max-width: 107%; }
+}
+
+/* Планшет */
+@media (max-width: 768px) {
+  .section-desc__wrap-down {
+    height: 538px;
+    background-position: 53% 16%;
+    background-size: 76%;
+  }
+
+  .safe-body__item_corps {
+    top: 75px;
+    right: 18px;
+  }
+
+  .item__icon_corps { max-width: 98%; }
+
+  .text__wrap_title-down {
+    padding-top: 59px;
+    margin-bottom: 10px;
+  }
+
+  .text__wrap_down { padding-left: 33px; }
+
+  .text__content_down { height: 163px; }
+
+  .text__wrap_character { 
+    justify-content: space-between;
+    width: 38%;
+  }
+  .text__wrap_character-down {
+    height: 238px;
+    margin-bottom: 0;
+    padding-top: 59px;
+    padding-left: 39px;
+  }
+
+  .text__wrap_subtitle { width: 100%; }
+
+  .text__subtitle { padding-left: 0; }
+
+  .text__wrap_section {
+    width: 100%;
+    height: 174px;
+    margin: 0;
+  }
+
+  .star__item {
+    width: 100%;
+    margin-right: 4px;
+  }
+
+  .score { display: none; }
+
+  .button_down { width: 70%; }
+
+  .button__item_big-down {
+    width: 40%;
+    padding-left: 0;
+    padding-right: 15px;
+  }
+}
+
+@media (max-width: 650px) {
+  .text__wrap_down,
+  .text__wrap_character { padding-left: 20px; }
+  .text__wrap_character {
+    width: 44%; 
+  }
+}
+
+@media (max-width: 600px) {
+  .section-desc__wrap { height: 415px; }
+
+  .text__wrap_section { height: 160px; }
+
+  .text__wrap_character-down { height: 220px; }
+
+  .star__item { 
+    margin-right: 0;
+    width: 20px;
+  }
+}
+
+@media (max-width: 500px) {
+  .section-desc__wrap { height: 321px; }
+
+  .safe-body__item_corps { top: 50px; }
+
+  .text__wrap_down,
+  .text__wrap_character { padding-left: 5px; }
+  .text__wrap_character { width: 47%; }
+}
+
+@media (max-width: 430px) {
+  .button_down { width: 80%; }
+}
+
+/* Телефон */
+@media (max-width: 375px) {
+  .section-desc__wrap-down {
+    height: 294px;
+    background-position: 0 22%;
+    background-size: 106%;
+  }
+
+  .safe-body__item_corps {
+    top: 57px;
+    right: 3px;
+  }
+
+  .item__icon_corps { max-width: 97%; }
+
+  .text__wrap_title-down {
+    padding-top: 15px;
+    margin-bottom: 9px;
+  }
+}
+
+@media (max-width: 320px) {
+  .text__title { padding-right: 0; }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)